ArduPlane/RC_Channel.cpp

```diff
@@ -277,6 +278,19 @@ void RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const AuxSwit
         }
         break;
 
+case AUX_FUNC::ARSPD_CALIBRATE:
```

Comment:
This will require the ` #if AP_AIRSPEED_AUTOCAL_ENABLE` pragma around it or it won't compile (the called function `set_calibration_enabled` is guarded by that same pragma.

Please test this code compiles if you change the enablement of that feature: https://github.com/ardupilot/ardupilot/blob/wip%2Fcsv-replay/libraries/AP_Airspeed/AP_Airspeed.h#L16

Reply:
done....